### ISSUE TITLE
Portal shouldn't crash

### ISSUE THREAD

**Issue by nickbradley** [ubccpsc/classy#212]
Just noticed that if I try to get a resource before authenticating, the following unhandled exception is thrown and the Portal service crashes.
```
<I> 2019-1-13 23:36:37: GeneralRoutes::getResource(..) - start
/app/packages/portal/backend/src/server/common/AdminRoutes.js:59
                const tokenParts = tokenString.split('__');
                                               ^

TypeError: Cannot read property 'split' of undefined
    at Function.processAuth (/app/packages/portal/backend/src/server/common/AdminRoutes.js:59:48)
    at Server.getResource (/app/packages/portal/backend/src/server/common/GeneralRoutes.js:122:44)
    at next (/app/node_modules/restify/lib/server.js:1126:29)
    at f (/app/node_modules/once/once.js:36:25)
    at Server.crossOrigin (/app/packages/portal/backend/src/server/BackendServer.js:53:24)
    at next (/app/node_modules/restify/lib/server.js:1126:29)
    at f (/app/node_modules/once/once.js:36:25)
    at Server.parseBody (/app/node_modules/restify/lib/plugins/bodyParser.js:152:17)
    at next (/app/node_modules/restify/lib/server.js:1126:29)
    at f (/app/node_modules/once/once.js:36:25)
    at f (/app/node_modules/once/once.js:25:25)
    at Server.readBody (/app/node_modules/restify/lib/plugins/bodyReader.js:87:13)
    at next (/app/node_modules/restify/lib/server.js:1126:29)
    at f (/app/node_modules/once/once.js:36:25)
    at Server.parseQueryString (/app/node_modules/restify/lib/plugins/query.js:53:20)
    at next (/app/node_modules/restify/lib/server.js:1126:29)
```

**Comment by rtholmes** [comment]
Was this on 310 or classy? I thought i fixed this a few days ago in #9d60a1

**Comment by rtholmes** [comment]
I think this will be fixed when we deploy next by #9d60a10.
